Dockerfile

```diff
@@ -1,31 +1,29 @@
-FROM debian:jessie
-MAINTAINER Luke Campbell <luke.campbell@rpsgroup.com>
+FROM node:13.7.0-alpine
+MAINTAINER RPS Dev-Ops <devops@rpsgroup.com>
```

Comment:
`MAINTAINER` is deprecated, it's recommended you use `LABEL` instead, per the official Dockerfile reference:  https://docs.docker.com/engine/reference/builder/#maintainer-deprecated